pkg/backend/httpstate/backend.go

```diff
@@ -274,17 +274,38 @@ func Login(ctx context.Context, d diag.Sink, cloudURL string, opts display.Optio
 	} else {
 		// If no access token is available from the environment, and we are interactive, prompt and offer to
 		// open a browser to make it easy to generate and use a fresh token.
-		line1 := fmt.Sprintf("Manage your Pulumi stacks by logging in.")
-		line1len := len(line1)
-		line1 = colors.Highlight(line1, "Pulumi stacks", colors.Underline+colors.Bold)
-		fmt.Printf(opts.Color.Colorize(line1) + "\n")
-		maxlen := line1len
-
-		line2 := "Run `pulumi login --help` for alternative login options."
-		line2len := len(line2)
-		fmt.Printf(opts.Color.Colorize(line2) + "\n")
-		if line2len > maxlen {
-			maxlen = line2len
+
+		lines := []string{
+			"",
+			colors.SpecHeadline + "Welcome to Pulumi!" + colors.Reset,
+			"",
+			"  Pulumi helps you create, deploy, and manage infrastructure on any cloud using",
+			"  your favorite language.  To manage your cloud infrastructure, Pulumi stores the",
+			"  state of your cloud deployments in a backend of your choosing.",
+			"",
+			colors.Highlight(
+				"  To use the default Pulumi Service backend, login below.  If you would prefer to",
+				"login below",
+				colors.BrightCyan+colors.Bold,
+			),
+			"  manage state yourself, you can choose from a variety of self-managed Pulumi",
+			"  backends. Run `pulumi login --help` for alternative login options.",
+			"",
+			colors.Highlight(
+				"  The Pulumi Service backend provides transparent, robust and simple storage for your",
+				"Pulumi Service",
+				colors.Underline+colors.Bold,
+			),
+			"  Pulumi state. All secret values are encrypted in the state, and you can",
+			"  bring your own encryption provider for secrets. Pulumi deployments are always executed",
+			"  locally where the `pulumi` CLI is being invoked, so no ambient cloud credentials,",
+			"  source code, or other sensitive data will ever be shared with the Pulumi Service",
+			"  backend. For more details - see https://www.pulumi.com/docs/intro/concepts/state/.",
```

Comment:
Seems like this might be worth highlighting, maybe bold, or green text, or similar?